### PR TITLE
Add auth to the mass-media console

### PR DESCRIPTION
**Pull request by PrPleGoo**
## About the PR
Stores the author's name and recordkeys on an article.
Articles can only be deleted by their authors.
If no author is found (based on ID card availability) the author is anonymous and anyone can delete it.

**Media**
- [X] this PR does not require an ingame showcase

**Changelog**
:cl:
- tweak: Changed news article to track an author.
- tweak: News articles can only be deleted by their tracked author.

### CODE REVIEW

File: Content.Client/MassMedia/Ui/NewsReadMenu.xaml.cs [hunk 1]
```diff
@@ -34,13 +30,13 @@ public void UpdateUI(NewsArticle article, int targetNum, int totalNum)
         PageText.Visible = true;
         ShareTime.Visible = true;
 
-        PageName.Text = article.Name;
+        PageName.Text = $"{article.Name} by {article.Author ?? "Anonymous"}";
```
Comment (EmoGarbage404) [inline Content.Client/MassMedia/Ui/NewsReadMenu.xaml.cs:33]: Make "Anonymous" a locale string

File: Content.Server/PDA/Ringer/RingerComponent.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ public sealed class RingerComponent : Component
         /// </summary>
         [ViewVariables(VVAccess.ReadWrite)]
         [DataField("range")]
-        public float Range = 0.5f;
+        public float Range = 3f;
```
Comment (EmoGarbage404) [inline Content.Server/PDA/Ringer/RingerComponent.cs:25]: You sure the increase wont lead to a lot of noise pollution?

**Review by EmoGarbage404 — APPROVED** [review]
Overall very fast and nice QOL

**Comment by Emisse** [comment]
agree on the locale comment from emo, otherwise lg2m

File: Content.Client/MassMedia/Ui/NewsReadMenu.xaml.cs (hunk 1 above)
Comment (PrPleGoo) [reply Content.Client/MassMedia/Ui/NewsReadMenu.xaml.cs]: fixed

File: Content.Server/PDA/Ringer/RingerComponent.cs (hunk 2 above)
Comment (PrPleGoo) [reply Content.Server/PDA/Ringer/RingerComponent.cs:25]: The previous PR reduced this from 3 to .5 and it got merged before the OG author could change it back.

**Comment by PrPleGoo** [comment]
WAIT

**Comment by PrPleGoo** [comment]
okay, go

**Pull request merged**